### PR TITLE
Fix broken links and various spelling things

### PR DESCRIPTION
**Pull request by RG-nAmKcAz**
- Fix broken links and various spelling things

---

Find the pull request instructions [here](https://www.notion.so/regressiongg/Pull-Request-Process-0d57a7eb582a446983edfacafa406f1e?pvs=4)

Every reviewer and the owner of the PR should consider these points in their request (feel free to copy this checklist so you can fill it out yourself in the overall PR comment)

- [ ] The code is extensible and backward compatible
- [ ] New public interfaces are extensible and open to backward compatibility in the future
- [ ] If preparing to remove a field in the future (i.e. this PR removes an argument), the argument stays but is no longer functional, and attaches a deprecation warning. A linear task is also created to track this deletion task.
- [ ] Non-critical or potentially modifiable arguments are optional
- [ ] Breaking changes and the approach to handling them have been verified with the team (in the Linear task, design doc, or PR itself)
- [ ] The code is easy to read
- [ ] Unit tests are added for expected and edge cases
- [ ] Integration tests are added for expected and edge cases
- [ ] Functions and classes are documented
- [ ] Migrations for both up and down operations are completed
- [ ] A documentation PR is created and being reviewed for anything in this PR that requires knowledge to use
- [ ] Implications on other dependent code (i.e. sample games and sample bots) is considered, mentioned, and properly handled
- [ ] Style changes and other non-blocking changes are marked as non-blocking from reviewers

### CODE REVIEW

File: docs/automated-testing/creating-custom-components.mdx [hunk 1]
```diff
@@ -190,6 +190,6 @@ We recommend creating a Prefab and attaching the Bot component to that prefab:
 ![A Unity Prefab showing the "My Bot (Script)" component added](img/creating-custom-components/bot-prefab.png)
 
 Then, you can add that Prefab to the `availableBots` property on the RegressionGames UI Overlay,
-which will add it to the dropdown of bots that can be spwaned in the UI overlay.
+which will add it to the dropdown of bots that can be spawned in the UI overlay.
 
-![A recording showing the user draggin the "MyBot" prefab on to the "Available Bots" list in the "Overlay Menu" section of a game object](img/creating-custom-components/recording-add-custom-bot-to-overlay.gif)
```
Comment (vontell) [inline docs/automated-testing/creating-custom-components.mdx:195]: Nothing wrong with being a little casual

**Review by vontell — APPROVED** [review]

**Review by abeizer — APPROVED** [review]
